### ISSUE TITLE
Jessie arm64 issue

### ISSUE THREAD

**Issue by pellet** [conan-io/conan#417]
```
It seems to be the first time you run conan                                                                      Auto detecting your dev setup to initialize conan.conf
Found gcc 4.9
Default conan.conf settings
        os=Linux
        arch=aarch64                                                                                                     compiler=gcc                                                                                                     compiler.version=4.9                                                                                             compiler.libcxx=libstdc++                                                                                        build_type=Release                                                                                       *** You can change them in ~/.conan/conan.conf ***
*** Or override with -s compiler='other' -s ...s***


ERROR: 'aarch64' is not a valid 'settings.arch' value.
Possible values are ['armv6', 'armv7', 'armv7hf', 'armv8', 'x86', 'x86_64']                                      -- The C compiler identification is GNU 4.9.2
-- The CXX compiler identification is GNU 4.9.2
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
CMake Error at CMakeLists.txt:4 (include):
  include could not find load file:

    build/conanbuildinfo.cmake


CMake Error at CMakeLists.txt:5 (conan_basic_setup):
  Unknown CMake command "conan_basic_setup".


-- Configuring incomplete, errors occurred!
```
Log should be enough to explain :)

**Comment by memsharded** [comment]
The issue is that the ``AArch64`` ARM architecture is not handled by conan yet in the detection step. From the settings point of view, I would say it is equivalent to the ``armv8`` existing conan setting one, can you please confirm it?

You can just go to your user home ``~/.conan/conan.conf`` and change the default settings architecture to  ``armv8``, and it shouldn't complain anymore, it seems it is just a problem of the automatic system detection.

Please tell me, so I can submit a PR to fix this misdetection. Thanks very much for reporting!

**Comment by Xaltotun** [comment]
armv8 and AArch64 are equivalent, but AArch64 is the canonical name for the instruction set architecture, since armv8 can run 32bit and 64bit architectures (AArch64, and AArch32).

**Comment by memsharded** [comment]
Thanks for the feedback @Xaltotun . The issue is that there is already an existing ``armv8`` setting in conan, changing it might break existing packages. We'd need to find some time and review ongoing work by @astralien3000 regarding settings and more specifically embedded settings, from issue https://github.com/conan-io/conan/issues/86.

**Comment by memsharded** [comment]
Also the ``ppc64le`` architecture from issue https://github.com/conan-io/conan/issues/405 should be added.

**Comment by pellet** [comment]
hi, thanks for the quick feedback-sorry i kinda forgot about the issue since I can confirm changing it to armv8 worked fine.  I wasn't aware that it was equivalent... that makes me feel happier, I thought that maybe armv8 was an older arch.
Possibly you could add detect aarch64 and just set it to armv8 with a warning message saying it is equivalent?
Thanks!

**Comment by memsharded** [comment]
Well, I am concerned about it, because if you can compile for armv8 but with 32bit, then that would be a different setting :(

If we assume that armv8 is the default AArch64, then maybe we could add armv8-32, or AArch32, to account for that setting. I would love to know if this makes sense, so I could add it to the detection step.

**Comment by pellet** [comment]
ok, not sure if armv8 defaults to aarch64 but if it does, i guess that sounds ok?

**Comment by pellet** [comment]
sounds good to me, 32 bit is on it's way out.

**Comment by memsharded** [comment]
Ok, so added the auto detection from ``AArch64`` => ``armv8``

**Comment by memsharded** [comment]
Released in conan 0.13.1
